client/boot/common.js

```diff
@@ -237,10 +238,27 @@ const configureReduxStore = ( currentUser, reduxStore ) => {
 };
 
 function setupErrorLogger( reduxStore ) {
+	// Add a bit of metadata from the redux store to the sentry event.
+	const beforeSend = ( event ) => {
+		const state = reduxStore.getState();
+		if ( ! event.tags ) {
+			event.tags = {};
+		}
+		event.tags.blog_id = getSelectedSiteId( state );
+		event.tags.calypso_section = getSectionName( state );
+		return event;
+	};
+
+	// Note that Sentry can disable itself and do some cleanup if needed, so we
+	// run it before the catch-js-errors check. (Otherwise, cleanup would never
+	// never happen.)
+	initSentry( { beforeSend } );
+
 	if ( ! config.isEnabled( 'catch-js-errors' ) ) {
 		return;
 	}
 
+	// At this point, the normal error logger is still set up so that
```

Comment:
Looks like this sentence needs to be finished after "so that..."?

Reply:
Good catch!